Admin.js

```diff
@@ -114,561 +136,771 @@ var Admin = {
      *
      * @param {string|Object} subject The html selector or object on which function should be applied
      */
-    setup_checkbox_range_selection: function(subject) {
-        Admin.log('[core|setup_checkbox_range_selection] configure checkbox range selection on', subject);
+    "setup_checkbox_range_selection" (subject) {
```

Comment:
- eslint: Expected object keys to be in ascending order. 'setup_checkbox_range_selection' should be before 'setup_icheck'.